packages/eventcatalog/hooks/EventCatalog.tsx

```diff
@@ -26,8 +26,11 @@ export const useUser = () => {
 
 export const useUrl = () => {
   const config = useConfig();
-
-  const getEditUrl = (url: string) => path.join(config.editUrl, url);
+  // eslint-disable-next-line no-unused-vars
```

Comment:
@boyney123 I'm not happy with this one, but I was not sure how to best tackle this.

Reply:
Hmm yeah let me take a quick look, also there might be another way with the hooks to do this